weather/components/LocationForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+
+import { Button, FormWrapper, Input } from "../devlink";
+
+const schema = yup
+  .object({
+    location: yup.string().required("Please enter a location."),
+  })
+  .required();
+
+export const LocationForm = () => {
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      location: "",
+    },
+    resolver: yupResolver(schema),
+  });
+
+  return (
+    <form onSubmit={handleSubmit((data) => console.log(JSON.stringify(data)))}>
+      <FormWrapper
+        formBody={
+          <>
+            <Controller
+              name="location"
+              control={control}
+              render={({ field }) => (
+                <Input
+                  isError={!!errors.location}
+                  errorMessage={errors.location?.message}
+                  inputProps={{
```

Comment:
Looks like we're missing this `inputProps` runtime prop in the `Input` component

<img width="240" alt="Screenshot 2023-02-21 at 16 00 32" src="https://user-images.githubusercontent.com/5628190/220366165-ac64d4de-aa3c-4af1-9f45-0e5924656e43.png">